src/status_im/ui/screens/wallet/choose_recipient/events.cljs

```diff
@@ -57,6 +61,27 @@
   (assoc-in fx [:db :wallet :send-transaction :gas]
             ethereum/default-transaction-gas))
 
+(re-frame/reg-fx
+ :resolve-address
+ (fn [{:keys [web3 registry ens-name cb]}]
+   (stateofus/addr web3 registry ens-name cb)))
```

Comment:
this doesn't look right, why stateofus care about .eth names

Reply:
you mean like why is it in this namespace at all?

Reply:
btw, `.eth` names can also add status whisper pubkey, so following that logic, there should be no `stateofus` ns